.github/workflows/rust.yml

```diff
@@ -108,14 +108,37 @@ jobs:
       run: |
         cargo install --path cargo-apk
         rustup target add x86_64-linux-android
+    - name: AVD cache
+      uses: actions/cache@v2
+      id: avd-cache
+      with:
+        path: |
+          ~/.android/avd/*
+          ~/.android/adb*
+        # One cache for 3 apk files
+        key: avd-emul
```

Comment:
No need to mention that this is for 3 apk files specifically, just that this key is not variable meaning it doesn't depend on the job type/target... Doesn't need a comment at all IMO.

Perhaps `avd-emulator` reads nicer though?